ext/openssl/openssl.c

```diff
@@ -3588,7 +3592,7 @@ static EVP_PKEY * php_openssl_generate_private_key(struct php_x509_request * req
 	EVP_PKEY * return_val = NULL;
 
 	if (req->priv_key_bits < MIN_KEY_LENGTH) {
-		php_error_docref(NULL, E_WARNING, "Private key length is too short; it needs to be at least %d bits, not %d",
+		zend_value_error("Private key length is too short; it needs to be at least %d bits, not %d",
```

Comment:
if you are changing this, you should change this one as well: https://github.com/php/php-src/blob/8c1fdd2385b5d8126c555d7d313a6a74e5d6c92b/ext/openssl/openssl.c#L3707